content/docs/custom-plugins/authenticated-proxy/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Configuring Authenticated Proxy for Custom Plugins
+lastUpdated: '2023-02-03T10:27:00.0Z'
+description: How to configure a custom proxy to identify the Backstage user for you custom plugin.
+---
+
+## Introduction
+
+Roadie provides a custom proxy service that can be configured to proxy through an access token minted by our own authentication system that can be validated by your third party service.
```

Comment:
Can you help me understand why I would want this?

Reply:
Ive added more text here. And a diagram.